web/src/main/webapp/js/map.js

```diff
@@ -164,12 +164,12 @@ function initMap(bounds, setStartCoord, setIntermediateCoord, setEndCoord, selec
         },
         contextmenu: true,
         contextmenuItems: [{
-                text: 'Route ',
```

Comment:
Added the space after "Route".

Same question about parametrized translation can be raised below for keys "start_label", "intermediate_label", "end_label", where a number is added after the translated string + space.

I don't think parametrized translation is necessary here.

Yes, all these keys were already added on the spreadsheet a few months ago.